src/components/elementView/elementTab.tsx

```diff
@@ -236,7 +236,7 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
       console.log("success");
       this.handleDataUpdate(true);
       this.handleClose();
-      this.addSuccessToast("Successfully created model!");
+      this.addSuccessToast(res.data.success_message);
```

Comment:
Did you confirm that there's a success_message returned in all the places you changed this?  I'm not sure every backend endpoint returns one

Reply:
yeah im 99% sure  -> these changes were made when julia worked on pretty error messages (which also included success messages). this is actually bad that we were hard coding this info because we were losing on information being sent from backend.